history/cultures/pandaren_group.txt

```diff
@@ -1,45 +1,64 @@
+# Mainland Pandaren (Sheili)
+# Wandering Pandaren (Shen-zin)
+
```

Comment:
Pandarens don't live in forests. Why do they have Guerilla Warfare?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/127783031-2feee966-380e-485c-93f2-c2637cf80b1a.png)

</details>